modules/space/rendering/renderableconstellationlines.cpp

```diff
@@ -0,0 +1,449 @@
+
+    createConstellations();
+}
+
+void RenderableConstellationLines::deinitializeGL() {
+    for (const std::pair<const int, ConstellationLine>& pair :
+        _renderingConstellationsMap)
+    {
+        glDeleteVertexArrays(1, &pair.second.vaoArray);
+        glDeleteBuffers(1, &pair.second.vboArray);
+    }
+
+    if (_program) {
+        global::renderEngine->removeRenderProgram(_program.get());
+        _program = nullptr;
+    }
+}
+
+void RenderableConstellationLines::renderConstellations(const RenderData&,
+                                                        const glm::dmat4& modelViewMatrix,
+                                                       const glm::dmat4& projectionMatrix)
+{
+    glEnablei(GL_BLEND, 0);
+    glBlendFunc(GL_SRC_ALPHA, GL_ONE);
+    glDepthMask(false);
+    glEnable(GL_DEPTH_TEST);
+
+    _program->activate();
+
+    _program->setUniform(_uniformCache.modelViewTransform, modelViewMatrix);
+    _program->setUniform(_uniformCache.projectionTransform, projectionMatrix);
+    _program->setUniform(_uniformCache.alphaValue, opacity());
+
+    for (const std::pair<const int, ConstellationLine>& pair :
+        _renderingConstellationsMap)
+    {
+        if (!pair.second.isEnabled) {
+            continue;
+        }
+
+        _program->setUniform(
+            _uniformCache.color,
+            _constellationColorMap[pair.second.colorIndex]
+        );
+
+        glBindVertexArray(pair.second.vaoArray);
+
+        glLineWidth(_lineWidth);
+        glDrawArrays(GL_LINE_STRIP, 0, pair.second.numV);
+        global::renderEngine->openglStateCache().resetLineState();
+    }
+
+    glBindVertexArray(0);
+    _program->deactivate();
+
+    // Restores GL State
+    global::renderEngine->openglStateCache().resetDepthState();
+    global::renderEngine->openglStateCache().resetBlendState();
+}
+
+void RenderableConstellationLines::render(const RenderData& data, RendererTasks& tasks) {
+    const glm::dmat4 modelMatrix =
+        glm::translate(glm::dmat4(1.0), data.modelTransform.translation) * // Translation
+        glm::dmat4(data.modelTransform.rotation) *  // Spice rotation
+        glm::scale(glm::dmat4(1.0), glm::dvec3(data.modelTransform.scale));
+
+    const glm::dmat4 modelViewMatrix = data.camera.combinedViewMatrix() * modelMatrix;
+    const glm::dmat4 projectionMatrix = data.camera.projectionMatrix();
+
+    if (_hasSpeckFile) {
+        renderConstellations(data, modelViewMatrix, projectionMatrix);
+    }
+
+    RenderableConstellation::render(data, tasks);
+}
+
+void RenderableConstellationLines::update(const UpdateData&) {
+    if (_program->isDirty()) {
+        _program->rebuildFromFile();
+        ghoul::opengl::updateUniformLocations(*_program, _uniformCache, UniformNames);
+    }
+}
+
+bool RenderableConstellationLines::loadData() {
+    bool success = false;
+    if (_hasSpeckFile) {
+        LINFO(fmt::format("Loading Speck file {}", std::filesystem::path(_speckFile)));
```

Comment:
Maybe no need for the `std::filesystem::path`?  Or alternatively changing the header variable to the `std::filesystem::path` type?